### ISSUE TITLE
ROI distribution over slices of all files

### ISSUE THREAD

**Issue by hdnminh** [MECLabTUDA/Lifelong-nnUNet#9]
Hi team,

I've done some code about exploring and visulizing official Prostate data. There are some weird distribution of ROI over slices of all files.

**1. UCL** 
![image](https://github.com/MECLabTUDA/Lifelong-nnUNet/assets/76483242/274774e6-60b3-4892-9b1f-19e1a775e4a2)
This seems they are following normal distribution.

**2. I2CVB**
![image](https://github.com/MECLabTUDA/Lifelong-nnUNet/assets/76483242/60d1bd57-150b-4c7b-9dbb-011c523a2e0e)
But with I2CVB, there is a file that I think this is a noise file. Do you think so?

**3. ISBI**
![image](https://github.com/MECLabTUDA/Lifelong-nnUNet/assets/76483242/ce5ebdad-d1b3-40c4-a330-a4e27e44f60b)
I think there is a noise file here.

**4. DecathProst**
![image](https://github.com/MECLabTUDA/Lifelong-nnUNet/assets/76483242/c2134f13-4834-4858-a8c0-87f98b2783ae)
 I think there are 2 noise files here.

To sum up, I remembered a quote that: "Garbage in Garbage out". I think there is something wrong with the data. How do you think?

**Comment by amrane99** [comment]
I am not sure how this relates to the Lifelong nnU-Net..
If you have actual questions about the Lifelong nnU-Net code, feel free to open relevant issues.
